### PR TITLE
Refactor : refactor md input parameters

### LINKED ISSUE #763: refactor md input parameters
The MD input parameters need to be refactored, the details are reported as follows:

1) `md_mdtype`  ->  `md_type` : control the ensemble to run md;
2) `md_potential`  ->  `md_ensolver` : choose the energy solver for MD;
3) `md_rstmd`  ->  `md_restart` : control whether restart md;
4) `md_dt`  ->  `md_dt` : time step(fs) used in md simulation;
5) `md_tfirst`  ->  `md_tfirst` : initial MD temperature;
6) `md_tlast`  ->  `md_tlast` : final MD temperature;
7) `md_dumpfred`  ->  `md_dumpfreq` : the frequence to dump md information;
8) `md_rstfred`  ->  `md_restartfreq` : the frequence to output restart information;
9) `md_tfreq`  ->  `md_tfreq` : Oscillation frequency, used to determine Qmass of NHC, or 1/(md_tfreq*md_dt) is collision probability in Anderson method;
10) `mnhc`  ->  `md_mnhc` : Number of Nose-Hoover chains;
11) `rcut_lj`  ->  `lj_rcut` : Cut-off radius for Leonard Jones potential (angstrom);
12) `epsilon_lj`  ->  `lj_epsilon` : The value of epsilon for Leonard Jones potential (eV);
13) `sigma_lj`  ->  `lj_sigma` : The value of sigma for Leonard Jones potential (angstrom);
14) `md_qmass`  ->  `msst_qmass` : Inertia of extended system variable for MSST;
15) `direction`  ->  `msst_direction` : the direction of shock wave for MSST;
16) `velocity`  ->  `msst_vel` : the velocity of shock wave (\AA/fs);
17) `viscosity`  ->  `msst_vis` : artificial viscosity (mass/length/time) for MSST;
18) `tscale`  ->  `msst_tscale` : reduction in initial temperature (0~1) used to compress volume in MSST;
19) `damp`  ->  `md_damp` : damping parameter (fs) used to add force in Langevin method.

### PR DESCRIPTION
**Pull request by YuLiu98**
The MD input parameters need to be refactored, the details are reported as follows:

1) `md_mdtype`  ->  `md_type` : control the ensemble to run md;
2) `md_potential`  ->  `md_ensolver` : choose the energy solver for MD;
3) `md_rstmd`  ->  `md_restart` : control whether restart md;
4) `md_dt`  ->  `md_dt` : time step(fs) used in md simulation;
5) `md_tfirst`  ->  `md_tfirst` : initial MD temperature;
6) `md_tlast`  ->  `md_tlast` : final MD temperature;
7) `md_dumpfred`  ->  `md_dumpfreq` : the frequence to dump md information;
8) `md_rstfred`  ->  `md_restartfreq` : the frequence to output restart information;
9) `md_tfreq`  ->  `md_tfreq` : Oscillation frequency, used to determine Qmass of NHC, or 1/(md_tfreq*md_dt) is collision probability in Anderson method;
10) `mnhc`  ->  `md_mnhc` : Number of Nose-Hoover chains;
11) `rcut_lj`  ->  `lj_rcut` : Cut-off radius for Leonard Jones potential (angstrom);
12) `epsilon_lj`  ->  `lj_epsilon` : The value of epsilon for Leonard Jones potential (eV);
13) `sigma_lj`  ->  `lj_sigma` : The value of sigma for Leonard Jones potential (angstrom);
14) `md_qmass`  ->  `msst_qmass` : Inertia of extended system variable for MSST;
15) `direction`  ->  `msst_direction` : the direction of shock wave for MSST;
16) `velocity`  ->  `msst_vel` : the velocity of shock wave (\AA/fs);
17) `viscosity`  ->  `msst_vis` : artificial viscosity (mass/length/time) for MSST;
18) `tscale`  ->  `msst_tscale` : reduction in initial temperature (0~1) used to compress volume in MSST;
19) `damp`  ->  `md_damp` : damping parameter (fs) used to add force in Langevin method.

### CODE REVIEW

**Comment by caic99** [comment]
Hi @YuLiu98 ,
Will you provide a detailed information on this change? If it is the same with issue contents, you can paste them here.
Besides, is there any scripts for users converting current input format to the new one? It seems that there's no backward compatiable in this version, and this change might make the former keys unavailable. cc @wenfei-li @ouqi0711

File: doc/examples/tddft.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ out_charge 1
 gamma_only 0
 nstep 2000
 dr2 1.0e-6
-md_mdtype 0
+md_type 0
 md_dt 0.01
 md_tfirst 30
 md_mdoutpath mdtest
```
Comment (dyzheng) [inline doc/examples/tddft.md]: this parameter is not used now，please delete it from document.

File: doc/examples/tddft.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ out_charge 1
...
-md_rstmd 0
+md_restart 0
 md_fixtemperature 20000
```
Comment (dyzheng) [inline doc/examples/tddft.md]: this parameter is not used now，please delete it from document.

File: doc/examples/tddft.md [hunk 3]
```diff
@@ -25,11 +25,11 @@ out_charge 1
...
 md_nresn 3
```
Comment (dyzheng) [inline doc/examples/tddft.md]: this parameter is not used now，please delete it from document.

File: doc/examples/tddft.md [hunk 4]
```diff
@@ -25,11 +25,11 @@ out_charge 1
...
 md_nyosh 3
```
Comment (dyzheng) [inline doc/examples/tddft.md]: this parameter is not used now，please delete it from document.

**Review by dyzheng — CHANGES_REQUESTED** [review]

**Comment by YuLiu98** [comment]
> Will you provide a detailed information on this change? If it is the same with issue contents, you can paste them here.
> Besides, is there any scripts for users converting current input format to the new one? It seems that there's no backward compatiable in this version, and this change might make the former keys unavailable. cc @wenfei-li @ouqi0711

OK, I attached the scripts for users converting current input format to the new one as below.

[input_update.zip](https://github.com/deepmodeling/abacus-develop/files/8167402/input_update.zip)

**Review by mohanchen — APPROVED** [review]
I am OK with these changes.

**Review by dyzheng — APPROVED** [review]

**Pull request merged**